docs/dev/table/udfs.md

```diff
@@ -297,7 +297,7 @@ optionally implemented. While some of these methods allow the system more effici
 - `merge()` is required for many batch aggreagtions and session window aggregations.
 - `resetAccumulator()` is required for many batch aggregations.
 
-All methods of `AggregateFunction` must be declared as `public`, not `static` and named exactly as the names mentioned above. The methods `createAccumulator`, `getValue`, `getResultType`, and `getAccumulatorType` are defined in the `AggregateFunction` abstract class, while others are contracted methods. In order to define a table function, one has to extend the base class `org.apache.flink.table.functions.AggregateFunction` and implement one (or more) `accumulate` methods. 
```

Comment:
"The method `accumulate` can be overloaded with different custom types and arguments and also support variable arguments."

->

"The method `accumulate` can be overloaded with different parameter types and supports variable arguments."